Library/SharedFunctions.swift

```diff
@@ -21,6 +21,21 @@ internal func userIsBacking(reward: Reward, inProject project: Project) -> Bool
     || (backing.reward == nil && backing.rewardId == nil && reward == Reward.noReward)
 }
 
+/**
+ Returns a reward from a backing in a given project
+
+ - parameter backing: A backing
+ - parameter project: A project
+
+ - returns: A reward
+ */
+
+internal func reward(from backing: Backing?, inProject project: Project) -> Reward {
```

Comment:
Ah nice 👍 

I suppose this could also be a computed `var` on `Project` like `backedReward: Reward?`.

That said - if nothing is backed, it'll return `Reward.noReward` which doesn't seem correct, so I think the return type should be optional for this?

Reply:
I copied and pasted this straight out of `ProjectPamphletViewModel`, but can update. I think ideally it would take a non-optional `Backing` and return a non-optional `Reward`.

Reply:
Also just realized I forgot to add tests for this 😅